### PR TITLE
Adding Meteor guide

### PR DESCRIPTION
**Pull request by adelevie**
TODO:
- ~~Add notes for MongoDB set up~~

### CODE REVIEW

File: content/apps/meteor.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+```yaml
+---
+applications:
+- name: hello-meteor
+  memory: 128M
+  instances: 1
+  domain: 18f.gov
+env:
+  FORCE_HTTPS: true
```
Comment (afeld) [inline content/apps/meteor.md]: Should this be under the `application`?

File: content/apps/meteor.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+You should to customize the application name.
+
+## Deploy
+
+Push using the [`cf-meteor-buildpack`](https://github.com/cloudfoundry-community/cf-meteor-buildpack).
+
+```
+$ cf push hello-meteor -b https://github.com/csterwa/cf-meteor-buildpack.git
+```
```
Comment (afeld) [inline content/apps/meteor.md]: How about adding this to the manifest?

https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#buildpack

File: content/apps/meteor.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+```
+$ cf create-service mongodb-new hello-meteor-mongo
+```
+
+Bind the service:
+
+```
+$ cf bind-service hello-meteor hello-meteor-mongo
+```
```
Comment (afeld) [inline content/apps/meteor.md]: Thoughts about doing this in the manifest?

https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html#services-block

File: content/apps/meteor.md (hunk 1 above)
Comment (adelevie) [inline content/apps/meteor.md]: What do you mean? Just remove an `s`?

File: content/apps/meteor.md (hunk 2 above)
Comment (adelevie) [inline content/apps/meteor.md]: Yeah yeah, totes.

File: content/apps/meteor.md (hunk 1 above)
Comment (afeld) [inline content/apps/meteor.md]: I guess I mean "should these two lines be indented two additional spaces?"

Comment (adelevie) [inline content/apps/meteor.md]: Should it? Would you mind just editing inline to fix this?

Comment (afeld) [inline content/apps/meteor.md]: Done!

**Comment by adelevie** [comment]
Gotta figure out the buildpack situation...

**Comment by afeld** [comment]
@adelevie I would transfer the fork to the 18F org and change the instructions to use that for now.

/cc https://github.com/cloudfoundry-community/cf-meteor-buildpack/issues/4

**Comment by adelevie** [comment]
Rebased it all.

**Pull request merged**